package com.rodrigodev.xgen.test.organization.packages.packagesForErrorsAreGeneratedFromErrorNames_specialCases.error_name1.error_name2.error___name3_5___;

import com.rodrigodev.xgen.test.organization.packages.packagesForErrorsAreGeneratedFromErrorNames_specialCases.error_name1.error_name2.ErrorName2Exception;

/**
* Autogenerated by XGen on June 30, 2015.
*/
public class Error___Name3_5___Exception extends ErrorName2Exception {

    protected Error___Name3_5___Exception(String message) {
        super(message);
    }
}
